SUMO-254678: Doc update Database apps migration V1 to V2 (#5009)
SumoLogic/sumologic-documentation: docs/integrations/amazon-aws/dynamodb.md

2. Click **Save**.

### Configure fields in Sumo Logic

> As part of the app installation process, the following fields will created by default:

Reviewer: ```suggestion
As part of the app installation process, the following fields will be created by default:
```

* `tablename`

> To learn more about setting up fields, refer to [Sumo Logic Fields](/docs/manage/fields).

Reviewer: ```suggestion
For information on setting up fields, see [Fields](/docs/manage/fields).
```

### Field Extraction Rule(s)

> Field Extraction Rule for CloudTrail Logs are automatically created for Database Application Components named as **AwsObservabilityDynamoDBCloudTrailLogsFER**.

Reviewer: ```suggestion
A Field Extraction Rule for CloudTrail Logs is automatically created for Database Application Components, named **AwsObservabilityDynamoDBCloudTrailLogsFER**.
```
